apps/oc_chef_wm/itest/setup_helper.erl

```diff
@@ -23,12 +23,21 @@
 -module(setup_helper).
 
 -include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
 
 -export([
          start_server/1,
-         needed_apps/0
+         needed_apps/0,
+         base_init_per_suite/1,
+         get_config/2
         ]).
 
+-define(TEST_DB_NAME, "oc_chef_wm_itests").
+
+-record(context, {reqid :: binary(),
```

Comment:
Instead of duplicating this  out of chef_db, use chef_db:make_context(<<"blah">>) when we need a context.